server/index.js

```diff
@@ -543,6 +543,34 @@ app.get('/api/posts/allUserPosts/:userId', (req, res, next) => {
     })
     .catch(err => next(err));
 });
+
+// delete chart
+
+app.delete('/api/comments/allCommentsToDelete/:commentId', (req, res, next) => {
+  const commentId = Number(req.params.commentId);
+
+  const sql = `
+  delete from "comments"
+  where "commentId"=$1
+  returning *
+  `;
+  const params = [commentId];
```

Comment:
You should also use the `req.user.userId` to prevent random clients from deleting any comment they want.